core/storage/file/gae_models.py

```diff
@@ -49,30 +49,87 @@ def get_new_id(cls, entity_name):
 
     @classmethod
     def get_undeleted(cls):
+        """Retrieve values that have not been deleted in the file metadata
+        model.
+
+        Returns:
+            A list of undeleted items from the query.
+        """
         return cls.get_all().filter(cls.deleted == False).fetch(  # pylint: disable=singleton-comparison
             feconf.DEFAULT_QUERY_LIMIT)
 
     @classmethod
     def _construct_id(cls, exploration_id, filepath):
+        """Builds the path for the id to be generated.
+
+        Args:
+            exploration_id: str. The ID of the exploration.
+            filepath: str. The path to the exploration file.
+
+        Returns:
+            The path of the id to be generated.
+        """
         return utils.vfs_construct_path('/', exploration_id, filepath)
 
     @classmethod
     def create(cls, exploration_id, filepath):
+        """Creates a new file meta data model entry.
+
+        Args:
+            exploration_id: str. The ID of the exploration.
+            filepath: str. The path to the exploration file.
+
+        Returns:
+            int. Instance of the new file meta data model entry.
```

Comment:
Yes, it's very important to check all the values properly -- please make sure you do this before submitting anything.